tgui/packages/tgui/interfaces/SpecMenu.js

```diff
@@ -150,8 +150,8 @@ const GarMenu = (props, context) => {
         </p>
         <p>
           <b>Overwhelming force</b>: Unlocked at 600 blood, when toggled, if you
-          bump into a door that you do not have access to, it will force it open.
-          In addition, you cannot be pushed or pulled while it is active.
+          bump into a door that you do not have access to, it will force it
```

Comment:
Here too.

Reply:
These are fine, they will go away as diffs once my PR is merged